frontend/src/app/modules/auth/register-form/register-form.component.ts

```diff
@@ -14,7 +16,8 @@ export class RegisterFormComponent {
   public hidePass = true;
   public hideConfirmPass = true;
 
-  constructor(private formBuilder: FormBuilder) {
+  constructor(private formBuilder: FormBuilder,
+    private registrationService: RegistrationService) {
```

Comment:
Rename className to AuthService

Reply:
If you know we use different service for Registration and for login, and I think in this case we shouldn't change name

Reply:
Ok, but there are some methods like log out, get token, check if token exist and this approach we will need third service to get this data, as it's not login or register